sklearn/metrics/_regression.py

```diff
@@ -480,14 +485,22 @@ def mean_squared_log_error(
     >>> y_pred = [2.5, 5, 4, 8]
     >>> mean_squared_log_error(y_true, y_pred)
     0.039...
+    >>> mean_squared_log_error(y_true, y_pred, squared=False)
+    0.199...
     >>> y_true = [[0.5, 1], [1, 2], [7, 6]]
     >>> y_pred = [[0.5, 2], [1, 2.5], [8, 8]]
     >>> mean_squared_log_error(y_true, y_pred)
     0.044...
+    >>> mean_squared_log_error(y_true, y_pred, squared=False)
+    0.210...
     >>> mean_squared_log_error(y_true, y_pred, multioutput='raw_values')
     array([0.00462428, 0.08377444])
+    >>> mean_squared_log_error(y_true, y_pred, multioutput='raw_values',squared=False)
+    array([0.06800207, 0.28943815]))
```

Comment:
Maybe remove this and the avobe example. 2 docstring examples of `squared=False` are enough IMO and we don't want to do all the combination of parameters ether here.

Reply:
Also we might take the opportunity to make this test more stable by using an ellipsis for the previous one:

```python
    >>> mean_squared_log_error(y_true, y_pred, multioutput='raw_values')
    array([0.004..., 0.083...])
```

Reply:
It's being failed again :(